stable/gangway/values.yaml

```diff
@@ -12,6 +12,13 @@ image:
 nameOverride: ""
 fullnameOverride: ""
 
+# Specify a CA cert to trust for self signed certificates at the Oauth2 URLs.  Be careful to indent one level beyond the
+# trustedCACert key:
```

Comment:
this like looks incorrect, possibly from previous commit?

Reply:
There should be a description for the new trustedCACert value.  What would you prefer the wording to be?

Reply:
🤦‍♂ ignore me. I'd not noticed it was the same comment as above it and it had wrapped!

Reply:
@rk295 There should be a description of the new trustedCACert value.  What would you prefer the wording to be?